verify-ui/src/components/Home/VerificationSection/Result/VcSvgTemplate.tsx

```
@@ -5,78 +5,72 @@ import { fetchSvgTemplate } from "../../../../utils/svg-template-utils";
       ADD_ATTR: ["target"],
       FORBID_TAGS: ["script", "iframe", "object", "embed"],
       FORBID_ATTR: ["onerror", "onload", "onclick", "onmouseover"],
     });
-  } catch (err) {
-    console.error("Mustache render error:", err);
     return (
-      <div className="text-red-500 text-sm text-center p-3">
-        Failed to render credential:{" "}
-        {err instanceof Error ? err.message : "Unknown error"}
+      <div className="w-full flex justify-center items-center">
+        <div dangerouslySetInnerHTML={{ __html: renderedSvg }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)